prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,33 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KM = 1.60934
+
+
+class ConvertMile(App):
+    """MilesConverterApp is a Kivy App for converting miles into kilometres"""
+    output_km = StringProperty()
+
+    def build(self):
+        """Build the Kivy app from the kv file."""
+        self.title = "Convert Miles to Kilometres"
+        self.root = Builder.load_file('convert_miles_km.kv')
```

Comment:
I think you need to add "self.output_km = '(km value)'" to show that you have used StringProperty().